### PR TITLE
Add my "Personal Highlights"

### PR DESCRIPTION
**Pull request by da5nsy**
### Summary

I thought I did this ages ago. Turns out, I did not!
Lovely to have a prompt to reflect - I've done some fun stuff!

### What should a reviewer concentrate their feedback on?

- I wasn't sure what formatting I should use for links here:
   - I've opted for this type when referring to people: `[Sarah Gibson](https://the-turing-way.netlify.app/afterword/contributors-record.html#sarah-gibson)` (but I feel like `[Sarah Gibson](https://github.com/sgibson91)` is also a reasonable choice...
   - And for links to the book I've used hard links, since I think that this file doesn't get processed by Jupyter Book, and so I don't think `{ref}`my-label-name`-type links will work (though - did this change since #3093 @JimMadge?)

### CODE REVIEW

File: contributors.md [hunk 1]
```diff
@@ -388,9 +388,28 @@ I also hope that it can convince people that don't consider themselves capable o
...
+> - [Learning how to build the book locally](https://web.archive.org/web/20221104150942/https://fosstodon.org/@da5nsy/108760121842868538) from [Sarah Gibson](https://the-turing-way.netlify.app/afterword/contributors-record.html#sarah-gibson) (and learning about miniconda in the process!) - a process which is now written up [here](https://the-turing-way.netlify.app/community-handbook/local-build.html#ch-local-build), but wasn't at the time, and so it felt like secret magic, and an infrastructure nerd's rite-of-passage.
+> - Using that knowledge to finally [wipe all of the long-standing build warning and errors](https://github.com/alan-turing-institute/the-turing-way/issues/2611) so that we could [implement CI to catch bugs before they were merged into main from then on](https://github.com/alan-turing-institute/the-turing-way/pull/2861)!
+> - [Presenting at FOSS Backstage in Berlin](https://github.com/alan-turing-institute/the-turing-way/issues/2993#issuecomment-1629424116) with [Anne Lee Steele](https://github.com/alan-turing-institute/the-turing-way/edit/main/contributors.md#anne-lee-steele), on the subject of transitioning from a reactive
```
Comment (KirstieJane) [inline contributors.md:411]: Is this sentence incomplete?

File: contributors.md [hunk 2]
```diff
@@ -388,9 +388,28 @@ I also hope that it can convince people that don't consider themselves capable o
...
+> - Using that knowledge to finally [wipe all of the long-standing build warning and errors](https://github.com/alan-turing-institute/the-turing-way/issues/2611) so that we could [implement CI to catch bugs before they were merged into main from then on](https://github.com/alan-turing-institute/the-turing-way/pull/2861)!
+> - [Presenting at FOSS Backstage in Berlin](https://github.com/alan-turing-institute/the-turing-way/issues/2993#issuecomment-1629424116) with [Anne Lee Steele](https://github.com/alan-turing-institute/the-turing-way/edit/main/contributors.md#anne-lee-steele), on the subject of transitioning from a reactive 
+> - Recently I'm focusing on the more organisation-level infrastructure ([what tools do we use and why](https://github.com/alan-turing-institute/the-turing-way/issues/3218)?), and as part of that I was proud to set up [the Turing Way's mastodon account](https://fosstodon.org/@turingway/108278354199514905).
```
Comment (KirstieJane) [inline contributors.md:412]: The "Recently" might not age well... what about "In 2023, I'm focusing..."

**Review by KirstieJane — COMMENTED** [review]
This is lovely @da5nsy!! Thank you so much!!

I'm reviewing from my phone so I can't check formatting or links.... so if someone else can check those that would be really helpful.

Content wise, it's an approve from me!

File: contributors.md (hunk 1 above)
Comment (da5nsy) [reply contributors.md]: Yep, just spotted that myself, whoops! Fixed in https://github.com/alan-turing-institute/the-turing-way/pull/3220/commits/52e9cd6e1b917187119cae668ef53288666d2caf 😅

File: contributors.md (hunk 2 above)
Comment (da5nsy) [reply contributors.md]: Done! 🎉

**Comment by da5nsy** [comment]
> I'm reviewing from my phone so I can't check formatting or links.... so if someone else can check those that would be really helpful.

I've just checked the links and switched to the `[Sarah Gibson](https://github.com/sgibson91)`-style links since they're a bit easier to read (I'd messed up the previous ones, partly because they were less legible in the other format 😅 )

File: contributors.md [hunk 3]
```diff
@@ -388,9 +388,28 @@ I also hope that it can convince people that don't consider themselves capable o
...
+* GitHub id: [da5nsy](https://github.com/da5nsy)
+* ORCID id: [0000-0002-0227-836X](https://orcid.org/0000-0002-4579-003X)
+* Fediverse profile: https://social.coop/@da5nsy
+* Website: https://www.dannygarside.co.uk/
+  
+* Short bio:
+ > I am a neuroscientist and meta-scientist, and I split my time between researching colour vision and trying to make academia more accessible, more efficient, and happier. 
+
+* Personal highlights:
+> - Learning about the project and [meeting Kirstie](https://github.com/alan-turing-institute/the-turing-way/blob/ebbaf8f5f3a66da441beafc690a1b16a9c50d15b/communications/newsletters/newsletter_08_Nov2019.md#acknowledgements-and-celebrations) during a rough patch in my PhD - learning about the project brought me hope for the future!
```
Comment (JimMadge) [inline contributors.md:406]:
```suggestion
> - Learning about the project and [meeting Kirstie](https://github.com/alan-turing-institute/the-turing-way/blob/ebbaf8f5f3a66da441beafc690a1b16a9c50d15b/communications/newsletters/newsletter_08_Nov2019.md#acknowledgements-and-celebrations) during a rough patch in my PhD — learning about the project brought me hope for the future!
```

**Review by JimMadge — APPROVED** [review]
Just being a pedant, won't block.

This was very nice, and quite heartwarming to read 🙂.

**Comment by KirstieJane** [comment]
@JimMadge - haha - those are some glorious suggested edits!

I'm merging!! Thank you @da5nsy!! For the great entry in the contributors list AND the awesome work! 💖

**Pull request merged**